### DISCUSSION TITLE
Https support

### DISCUSSION THREAD

**Discussion by susanta-mukhopadhyay** [9tigerio/db2rest#221] [General]
Are we supporting https or only http at present? If we support https, is there any guide to configure it?

**Comment by kdhrubo** [comment]
We must support both. Esp HTTPS. We need to test and document Https. 

However the software comes prepackaged as jar (native executable in near future) and docker image we have to test how to load certs from external source.

**Reply by susanta-mukhopadhyay** [reply]
We may think of having docker secret store and providing scripting option to load the certificate. As per default standard, we may provide self signed certificate to be used for development, but not recommended in production. What do you think?
